Makefile

```diff
@@ -5,8 +5,9 @@ CONTROLLER_TOOLS_VERSION=$(shell awk '/sigs\.k8s\.io\/controller-tools/ {print s
 CSI_VERSION=1.5.0
 PROTOC_VERSION=3.15.0
 KIND_VERSION=v0.11.1
-HELM_VERSION=3.5.0
-HELM_DOCS_VERSION=1.5.0
+HELM_VERSION=3.7.1
+HELM_DOCS_VERSION=1.6.0
```

Comment:
Could you please revert Helm Doc version to 1.5.0 because there are no v1.6.0 on the releases page when I checked? I guess it was canceled because the tag exists.

Reply:
reverted.